.github/release-drafter.yml

```diff
@@ -9,11 +9,22 @@ categories:
       - 'bugfix'
       - 'bug'
   - title: '🧰 Maintenance'
-    label: 'chore'
+    label: 
+      - 'chore'
+      - 'dependencies'
+  - title: '📖 Documentation'
+    label: 
+      - 'documentation'
+
 exclude-labels:
   - 'skip-release-drafter'
 change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
 template: |
   ## Changes
 
   $CHANGES
+
+ ## Contributors
```

Comment:
Not sure this is needed since release drafter is already doing `'- $TITLE @$AUTHOR (#$NUMBER)'` Which puts the author (aka contributor) in the change.

Reply:
Okay we can remove that

Reply:
Should I remove it @JamesSingleton

Reply:
I would remove the `## Contributors` section

Reply:
Yes I removed it